gallery/src/gallery-components/GalleryTiles.tsx

```diff
@@ -36,13 +46,28 @@ export const GalleryTile: FunctionComponent<GalleryTileProps> =
     );
   };
 
-// GalleryTile and GalleryExample components are just GalleryTiles with specified titles
+// GalleryDescriptionTile is just a GalleryTile with a specified title
 export const GalleryDescriptionTile: FunctionComponent<PropsWithRequiredChildren> =
   function GalleryDescriptionTile(props: PropsWithRequiredChildren) {
     return <GalleryTile { ...props } title="Description" className="gallery-description" />;
   };
 
-export const GalleryExampleTile: FunctionComponent<PropsWithRequiredChildren> =
-  function GalleryExampleTile(props: PropsWithRequiredChildren) {
-    return <GalleryTile { ...props } title="Example" className="gallery-example"/>;
+export const GalleryExampleTile: FunctionComponent<GalleryExampleTileProps> =
+  function GalleryExampleTile({ children, className, title, description, codeExamples }: GalleryExampleTileProps) {
+    const codeExampleElements = ensureArray(codeExamples)
+        .map((example, idx) => {
+          const props = typeof example === 'string' ? { content: example } : example;
+          return <CodeExample key={idx} { ...props } />;
+        });
+
+    return (
+      <GalleryTile title={title} className={className}>
+        <p className="nx-p">{description}</p>
+
+        <h3 className="nx-h3 nx-tile__section-header">Example:</h3>
+        <div>{children}</div>
```

Comment:
I think this should have the className of `.gallery-example`, which you have on the `<section>` element below.

Reply:
> I think this should have the className of .gallery-example, which you have on the <section> element below.

Why? All `.gallery-example` does is set the font size in the code snippets.  This `<div>` does not contain code snippets.

Reply:
> Why? All .gallery-example does is set the font size in the code snippets. This <div> does not contain code snippets.

I guess I had originally moved it because I thought it was making the description paragraph text small.  But it's not the cause there.  So I moved it back up to the tile itself in fa1d62f9eb and 7865b104c3